src/rust/src/properties_reader.rs

```diff
@@ -70,10 +75,17 @@ impl GeozeroGeometry for FgbFeature {
 impl geozero::FeatureProperties for FgbFeature {
     /// Process feature properties.
     fn process_properties<P: PropertyProcessor>(&self, reader: &mut P) -> Result<bool> {
-        let columns_meta = self
-            .header()
-            .columns()
-            .ok_or(GeozeroError::GeometryFormat)?;
+        let columns_meta_res_base: Option<Vector<ForwardsUOffset<Column>>> =
```

Comment:
note to self - this fixed one of the many opaque "GeometryFormat" errors that came from reading with the rust impl